src/indexer/index.ts

```diff
@@ -254,6 +259,30 @@ export function decodeAppParamsState(state: any): Map<Key, StateValue> {
 	return stateMap;
 }
 
+/**
+ * Decode local app params state
+ * @param state State to be decoded
+ */
+export function decodeLocalAppParamsState(state: any): Map<Key, StateValue> {
```

Comment:
Don't we have a function to decode local and global state, can't that be used here?
If not can we modify existing one to use it dynamically